indico/modules/users/templates/dashboard.html

```diff
@@ -28,7 +28,7 @@
 {%- endblock %}
 
 {% block banner_actions -%}
-    <div id="dashboard-calendar-link"></div>
+    <div id="dashboard-calendar-link" title="{% trans %}Export{% endtrans %}"></div>
```

Comment:
Does this still stay when the `ICSCalendarLink` component gets rendered into this container? I thought the container gets replaced but maybe it's just its contents...

Reply:
FWIW if you include the title inside the react component you can omit it in all the places where it gets rendered (e.g. `indico/modules/categories/templates/display/base.html` as well)

Reply:
> Does this still stay when the `ICSCalendarLink` component gets rendered into this container? I thought the container gets replaced but maybe it's just its contents...

Yes, it remains the same